go/cmd/dolt/commands/sql.go

```diff
@@ -235,7 +235,11 @@ func (cmd SqlCmd) Exec(ctx context.Context, commandStr string, args []string, dE
 			}
 			return HandleVErrAndExitCode(errhand.VerboseErrorFromError(ErrMultipleDoltCfgDirs.New(p1, p2)), usage)
 		}
-		cfgDirPath = path
+
+		// Only use current directory if there wasn't one in parent
```

Comment:
This isn't right, this is exactly opposite of the intended behavior

If you don't specify a doltcfg dir or a data dir, you should get the default, which is 1) ./.doltcfg if it exists, or 2) ../.doltcfg otherwise